app/decorators/casa_case_decorator.rb

```diff
@@ -1,6 +1,12 @@
 class CasaCaseDecorator < Draper::Decorator
   delegate_all
 
+  def status
+    return "Active" if object.active
+
+    "Inactive"
+  end
```

Comment:
I think the ternary operator might be a little nicer here:

```
def status
  object.active ? "Active" : "Inactive"
end
```

Reply:
Fixed